operations/services.md

```diff
@@ -272,6 +272,188 @@ Returns all products offered together with the specified services.
+        "ResourceCategories": true,
+        "BusinessSegments": true
+    }
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceIds` | array of string | required | Unique identifiers of the [Service](#service)s. |
+| `Extent` | [Rule extent](#rule-extent) | required | Extent of data to be returned. |
+
+#### Rule extent
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `RuleActions` | bool | optional | Whether the response should contain rule actions. |
+| `Rates` | bool | optional | Whether the response should contain rates. |
+| `RateGroups` | bool | optional | Whether the response should contain rate groups. |
+| `ResourceCategories` | bool | optional | Whether the response should contain rate resource categories. |
+| `BusinessSegments` | bool | optional | Whether the response should contain business segments. |
+
+
+### Response
+
+```javascript
+{
+    "Rules": [
+        {
+            "Id": "13638b12-53f1-4b35-baab-ac1e006ed8cb",
+            "Conditions": {
+                "RateId": {
+                    "Value": "8a13f4e8-0274-4bcc-b2d4-ac1c00d1cd3f",
+                    "ConditionType": "Equals"
+                },
+                "RateGroupId": null,
+                "BusinessSegmentId": null,
+                "ResourceCategoryId": null,
+                "ResourceCategoryType": null,
+                "Origin": {
+                    "Value": "Connector",
+                    "ConditionType": "NotEquals"
+                },
+                "MinimumTimeUnitCount": null,
+                "MaximumTimeUnitCount": null
+            }
+        }
+    ],
+    "RuleActions": [
+        {
+            "Id": "307e75a6-2cc8-4226-a600-ac1e006fbdb9",
+            "RuleId": "13638b12-53f1-4b35-baab-ac1e006ed8cb",
+            "Data": {
+                "Discriminator": "Product",
+                "Value": {
+                    "ProductId": "122fd92d-c561-4995-8ebc-ac1c00d1eaa8",
+                    "ActionType": "Add"
+                }
+            }
+        }
+    ],
+    "Rates": [
+        {
+            "Id": "8a13f4e8-0274-4bcc-b2d4-ac1c00d1cd3f",
+            "GroupId": "e4a9d8d1-5793-4d35-954e-ac1c00d1eaa8",
+            "ServiceId": "ea80bbca-372f-4550-8e48-ac1c00d1cd20",
+            "BaseRateId": null,
+            "IsActive": true,
+            "IsPublic": true,
+            "Name": "Fully Flexible",
+            "ShortName": "FF",
+            "ExternalNames": {}
+        }
+    ],
+    "RateGroups": [],
+    "ResourceCategories": [],
+    "BusinessSegments": []
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Rules` | array of [Rule](#rule) | required | Rules applied with reservations. |
```

Comment:
I'd improve the sentence here since "applied with reservations" is not that clear.